content/articles/2024-11-01_Version_0.9.6.md

```diff
@@ -0,0 +1,69 @@
+streamlining and enhancing the developer experience with several key updates, new features, and some fixes.
+
+This release introduces Docker-only support, ensuring a consistent and reliable deployment environment, along with 
+advanced Kubernetes namespace management capabilities to better handle scoped tenants and clients. Developers will also 
+benefit from new tooling commands, making it easier to launch Visual Studio Code in a browser within Docker and to 
+quickly clean up resources and maintain an optimized workspace.
+
+Additionally, this update brings important changes to improve performance and compatibility across various components, 
+from more efficient HTTP handling to updated namings. Core frameworks like Vapor and Swift have been upgraded, keeping 
+Uitsmijter compatible with current technologies and best practices. 
+
+With a host of improvements, Uitsmijter 0.9.6 is designed to offer a smoother, more efficient, and developer-friendly 
+experience on the way to a official version 1.0, that will be 🥁fully Swift 6 compatible. 
+
+### Here's what’s new:
+
+**Feature Additions**
+- **Namespace Scoped Tenants and Clients**: Uitsmijter now supports local namespace-scoped tenants and clients, making it easier to define boundaries and permissions specific for various projects working on one cluster.
+- **New Tooling Commands**: Two new commands have been added to streamline workflows:
+    - **`code`**: Launches Visual Studio Code in a Dockerized environment directly within your browser, allowing seamless code editing without setup. [Making it easier for new developers to get started with open source projects](./2024-03-06_code).
+    - **`remove`**: Efficiently cleans up your Docker environment by removing all Docker resources and clearing the build folder.
+
+**Support for Docker-only installations**  
+Starting with version 0.9.6, Uitsmijter will run seamlessly in a Docker only environment, too, while ensuring the same 
+functionallity consistent with the kubernetes installation. 
+This finally simplifies setups, reduces dependency conflicts, and makes Uitsmijter easier to use for smaller systems.
+
+**Key Changes**
+- **Scripting Provider’s Fetch Method**: The `fetch` method now leverages `AsyncHTTPClient` rather than `FoundationNetwork`'s `URLSession`, providing a more efficient and reliable way to handle HTTP requests.
+- **Redis Database Naming**: Redis instances are now clearly identified with `uitsmijter-session-master` and `uitsmijter-session-slave`, helping to avoid confusion and enhancing configuration clarity.
+- **End-to-End Testing in `--fast` Mode**: End-to-end tests running in `--fast` mode will now default to using Chromium, ensuring consistent behavior and results across tests.
+- **Default Login Page Image Support**: To streamline compatibility, we’ve removed `webp` support for the Uitsmijter default login page.
```

Comment:
Can you elaborate on this? The format is fairly widely supported, isn't it? https://caniuse.com/webp